vuepress/docs/next/tutorials/create/mfe/communication.md

```diff
@@ -76,37 +59,33 @@ customElements.define('publisher-widget', PublisherWidgetElement);
 export default PublisherWidgetElement;
 ```
 
-- In the `CustomEvent` constructor, `detail` is the specific name to use in the event payload, as per the [DOM specification](https://dom.spec.whatwg.org/#interface-customevent).
-
-#### Import Custom Element
+   In the `CustomEvent` constructor, `detail` denotes the specific name to use in the event payload per the [DOM specification](https://dom.spec.whatwg.org/#interface-customevent).
 
-Update `publisher-widget/src/index.js`.
+2. To import the custom element, update `publisher-widget/src/index.js`:
```

Comment:
no. update means replace the contents. in other tutorials we've edited this language to make that explicit but i feel like the intent is understood so i left as-is. to expand upon it seems unnecessary because if you're familiar with the file content update=replace should be obvious

Reply:
@jyunmitch struggling with this one... this tutorial is identified as intermediate wrt our learning paths... i asked nathan to weigh in